src/App.js

```diff
@@ -1,5 +1,99 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import Car from "./Car.js";
+import { validateCarNameList, validateTryCount } from "./validation.js";
+import {
+  CAR_NAME_INPUT_MESSAGE,
+  TRY_COUNT_INPUT_MESSAGE,
+} from "./constants/inputMessage.js";
+import {
+  EMPTY_MESSAGE,
+  EXECUTE_RESULT_MESSAGE,
+  WINNER_MESSAGE,
+} from "./constants/outputMessage.js";
+
 class App {
-  async run() {}
+  #tryCount;
+  #cars;
+
+  constructor() {
+    this.#tryCount = 0;
+    this.#cars = [];
+  }
+
+  async run() {
+    let carNameList = await this.readCarName();
+    validateCarNameList(carNameList);
+    this.generateCars(carNameList);
+
+    const tryCount = await this.readTryCount();
+    validateTryCount(Number(tryCount));
+    this.#tryCount = Number(tryCount);
+
+    this.printGameStart();
+    this.generateGame();
+
+    const winner = this.generateGameResult();
+    this.printGameResult(winner);
+  }
+
+  async readCarName() {
+    return await Console.readLineAsync(CAR_NAME_INPUT_MESSAGE);
+  }
+
+  async readTryCount() {
+    return await Console.readLineAsync(TRY_COUNT_INPUT_MESSAGE);
+  }
```

Comment:
콘솔 입력 받는거 말고 별도의 로직이 없긴 때문에 `async/await` 키워드가 의미가 없어서 이렇게 축약할 수 있을 것 같아요🙃
```suggestion
  readCarName() {
    return Console.readLineAsync(CAR_NAME_INPUT_MESSAGE);
  }

  readTryCount() {
    return Console.readLineAsync(TRY_COUNT_INPUT_MESSAGE);
  }
```

Reply:
습관처럼 async와 await를 사용했네요. 감사합니다!